doc/source/management/data_at_rest_encryption.rst

```diff
@@ -258,22 +258,46 @@ System Variables
   :vartype: Boolean
   :default: ``Off``
 
-When this option is turned on, server starts to encrypt temporary tablespace
-and temporary |InnoDB| file-per-table tablespaces. The option does not force
-encryption of temporary tables which are currently opened, and it doesn't
-rebuild system temporary tablespace to encrypt data which are already written.
+When this option is set to ``ON``, the server encrypts the temporary
+tablespace. This option does not enforce the encryption of temporary
+tables which are currently open, and it does not rebuild the system
+temporary tablespace to encrypt data which are already written.
+
+The ``ENCRYPTION`` option is not allowed in the ``CREATE TEMPORARY TABLE``
+statement. The ``TABLESPACE`` option cannot be set to `innodb_temporary`. The
+global temporary tablespace datafile ``ibtmp1`` holds temporary table undo logs
+while intrinsic temporary tables and user temporary tables go to the encrypted
+session temporary tablespace.
+
+Since the global temporary tablespaces (:file:`ibtmp*` data files) and session
```

Comment:
Remove the elaborate wording here. Just say "global temporary and session temporary tablespaces are"

Reply:
Ready for review